BrainPortal/app/controllers/application_controller.rb

```diff
@@ -69,6 +70,16 @@ class ApplicationController < ActionController::Base
 
   private
 
+
+  def redirect_show_license
+    #
```

Comment:
Is it a missing comment or an extr `#` you forgot to remove ?

Reply:
deleted